english/invite-friends/smores/after_smores/indigestion.md

```diff
@@ -0,0 +1,7 @@
+Seating back after those satisfying smores, you suddenly get very tired. 
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.